rbd-target-api.py

```diff
@@ -543,11 +543,19 @@ def _gateway(gateway_name=None):
 def get_disks():
     """
     Show the rbd disks defined to the gateways
+    :param detail: (str) 'yes' to list details of all disks, default is 'no'
     **RESTRICTED**
+    Examples:
+    curl --insecure --user admin:admin -d detail=yes -X GET https://192.168.122.69:5001/api/disks
```

Comment:
Thanks for the patches.

I am going to change the variable name from detail to config to reflect it will return the disk's config info.

Also, I am going to change the port from 5001 to 5000 to match the other examples you added.

Reply:
Hey Mike,

I have update all of these.

Thanks,